src/Components/Wizard/steps/FinishProgress/index.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  Title,
+  Progress,
+  Button,
+} from '@patternfly/react-core';
+import { CogsIcon, OkIcon, ErrorCircleOIcon } from '@patternfly/react-icons';
+import { useWizardContext } from '../../../Common/WizardContext';
+import { useMutation } from 'react-query';
+import { createAWSReservation, createNewPublicKey } from '../../../../API';
+import './styles.scss';
+
+const pf_success_color_100 = '#3E8635';
+const pf_danger_color_100 = '#C9190B';
+
+const FinishStep = ({ onClose, imageID }) => {
+  const [
+    {
+      chosenSource,
+      chosenInstanceType,
+      chosenNumOfInstances,
+      sshPublicName,
+      sshPublicKey,
+      chosenSshKeyId,
+    },
+  ] = useWizardContext();
+  const [, setReservationID] = React.useState();
+  const [activeStep, setActiveStep] = React.useState(chosenSshKeyId ? 1 : 0);
```

Comment:
Can we use the `uploadedKey`? I know we didn't decided on the usage, but I think we should be consistent and make a change on all the places once we decide to use one or the other :)
```suggestion
  const [activeStep, setActiveStep] = React.useState(uploadedKey ? 1 : 0);
```